{"name": "testcaseadd3[【类型】输入英文，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：30和t开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761870, "stop": 1661497761870}], "attachments": [{"name": "log", "source": "7dd7aa87-6394-43b0-b042-51509a5a882f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "30"}, {"name": "y", "value": "'t'"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761869, "stop": 1661497761870, "uuid": "8911ef87-c066-4e60-b02c-ed73973c9d2b", "historyId": "3b7c658c6967e558972579fd2cd4b2b9", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}